### ISSUE TITLE
Tooltip missing on several buttons in feed

### ISSUE THREAD

**Issue by joho1968** [pixelfed/pixelfed#4040]
There seems to be a few "buttons" or "clickables" that have no onmouseover tooltip, like the "boost" and bookmark icons under a post. Moderation tools does have a tooltip.

The filter icon and the reload icon in the right hand side notifications are also missing tooltips.

I see this is mentioned in #116, but maybe these icons are new or the design is new.
